nemo_curator/datasets/doc_dataset.py

```diff
@@ -37,6 +37,21 @@ def __len__(self) -> int:
     def persist(self) -> "DocumentDataset":
         return DocumentDataset(self.df.persist())
 
+    def compute(self) -> "DocumentDataset":
+        return DocumentDataset(self.df.compute())
```

Comment:
If we do this, then the `df` becomes a `pd.DataFrame / cudf.DataFrame` type, isn't?  While `self.df` is expected to be `dd.DataFrame`?

Reply:
Oh yeah that's fair. I'll remove this, mainly just thought it would be nice to have the `repartition` function.